website/content/api-docs/catalog.mdx

```diff
@@ -537,9 +537,16 @@ The table below shows this endpoint's support for
 - `filter` `(string: "")` - Specifies the expression used to filter the
   queries results prior to returning the data.
 
+- `merge-central-config` - If set on the request, the response will include a fully resolved service
+  definition which is merged with the [proxy-defaults/global](/docs/connect/config-entries/proxy-defaults) 
+  and [service-defaults/:service](/docs/connect/config-entries/service-defaults) config entries. This can be useful to view
+  the full service definition for a mesh service (connect-proxy kind or gateway kind) which might not be merged before being
+  registered into the catalog.
+
```

Comment:
It still is a bit off. How about:

``` 
When included in a request, the response contains a fully resolved service definition in case the service is a `connect-proxy` kind or `*-gateway` kind. The service definition is merged with the [proxy-defaults/globalm (/docs/connect/config-entries/proxy-defaults) and [service-defaults/:service](/docs/connect/config-entries/service-defaults) config entries. Returning a fully merged service definition is useful when a service is directly registered using the [consul catalog apis](https://www.consul.io/api-docs/catalog#register-entity) which does not automatically merge the config entries. 
```

Reply:
Okay, that sounds perfect to me. Will update. Thanks for the all the pointers.

Reply:
Updated!